pkg/cli/sync.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/abcxyz/pkg/cli"
+)
+
+var (
+	_                        cli.Command = (*SyncCommand)(nil)
+	allowedSourceSystem                  = []string{"GOOGLEGROUPS"}
+	allowedDestinationSystem             = []string{"GITHUB"}
+)
+
+type SyncCommand struct {
+	cli.BaseCommand
+
+	source             string
+	destination        string
+	groupMappingConfig string
+	userMappingConfig  string
+	sourceToken        string
+	destinationToken   string
+}
+
+func (c *SyncCommand) Desc() string {
+	return `Sync membership information`
+}
+
+func (c *SyncCommand) Help() string {
+	return `
+Usage: {{ COMMAND }} [options]
+
+  Sync membership information from source and target system.
+
+  Sync membership from google group to GitHub
+
+  tlctl sync run \
+	-src GoogleGroup \
+	-dest GitHub \
```

Comment:
Help string args should be updated to reflect current names/aliases, e.g. "dest" -> "dst"

Reply:
Also are auth tokens required for source and destination systems? If so the example command should include those flags

Reply:
Updated. Thank you for catching this!